inscriptions/ISic003203.xml

```diff
@@ -189,14 +194,24 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="unsupplied" xml:lang="la">
-                <ab><!--Greek or Latin (etc.) text here--></ab>
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#FP">
+                <ab>
+                    <lb n="1"/><gap reason="lost" extent="unknown" unit="character"/> <persName type="attested"><name type="nomen"><supplied reason="lost">Ma</supplied>nliae</name></persName> <persName type="attested"><name type="praenomen"><expan><abbr>T</abbr><ex>iti</ex></expan></name></persName> <w><expan><abbr>f</abbr><ex>iliae</ex></expan></w>
+                    <lb n="2"/><supplied reason="lost"><w>et</w></supplied> <persName type="attested"><supplied reason="lost"><name type="praenomen">Marci</name></supplied> <name type="nomen"><supplied reason="lost">Ro</supplied>sci</name> <persName type="attested"><name type="praenomen"><expan><abbr>M</abbr><ex>arci</ex></expan></name></persName> <w><expan><abbr>f</abbr><ex>ili</ex></expan></w> <orgName type="tribus" nymRef="#Quirina"><expan><abbr>Qui</abbr><ex>rina</ex></expan> <supplied reason="subaudible"><w lemma="tribus">tribu</w></supplied></orgName> 
```

Comment:
also here, on M(arci), type is patronym not preanomen.
I know it is in the example on the Wiki from Simona, but at the moment, on the tribal, just put the `<orgName>` element, but do not add the nymref, because we do not yet have an authority list for this. also, there is no need to put the lemma on the w for tribus - we will add all lemmatisation automatically afterwards.